test-framework/src/main/java/org/apache/brooklyn/test/framework/TestHttpCall.java

```diff
@@ -51,6 +51,8 @@
             .defaultValue(HttpMethod.GET)
             .build();
 
+    ConfigKey<Boolean> TRUST_ALL = ConfigKeys.newBooleanConfigKey("trustAll","Trust all certificates used to sign this request",true);
```

Comment:
Yes, I preferred to stick to the nomenclature defined by `HttpTool` and I'm not sure there's a lot else it could potentially be confused with.